### PR TITLE
docs: update Pro manpage

### PR DESCRIPTION
**Pull request by lucasmoura**
## Why is this needed?
Update the commands listed in the Pro manpage while also updating the services we support there as well

## Test Steps
Check if the `man pro` outputs the new manpage

---

- [ ] *(un)check this to re-run the checklist action*

### CODE REVIEW

File: ubuntu-advantage.1 [hunk 1]
```diff
@@ -171,176 +275,120 @@ Makes sure that the MOTD and APT messages match the available/enabled services
...
-.TP
-.B
-\fBlog_level\fP
-The logging level used when writing to \fBlog_file\fP
-.TP
-.B
-\fBlog_file\fP
-The log file for the Ubuntu Pro client cli
+\fBmetering\fP
+If attached, this job will ping the Canonical servers telling which
```
Comment (dheyay) [inline ubuntu-advantage.1:281]: "which" appears twice here

File: ubuntu-advantage.1 [hunk 2]
```diff
@@ -171,176 +275,120 @@ Makes sure that the MOTD and APT messages match the available/enabled services
...
+evaluated to assurance level EAL2 through CSEC. The evaluation was performed
+on Intel x86_64, IBM Power8 and IBM Z hardware platforms.
 
-The artifacts include a configure script, a tarball with additional
-packages, and post install scripts. The artifacts will be installed in
-/usr/lib/common-criteria directory and the README and configuration
-guide are available in /usr/share/doc/ubuntu-commoncriteria directory.
+.TP
+.B "CIS Audit (cis)/Ubuntu Security Guide (usg)"
+Ubuntu Security Guide is a tool for hardening and auditing and allows for
```
Comment (dheyay) [inline ubuntu-advantage.1:307]: I think we can reduce the third `and` in this sentence by using something like
```suggestion
Ubuntu Security Guide is a tool for hardening and auditing, allowing for
```

**Review by dheyay — CHANGES_REQUESTED** [review]
Thanks for this @lucasmoura!

Left a few comments with suggested changes (really small ones), otherwise the content looks great

**Comment by lucasmoura** [comment]
@dheyay updated

**Review by dheyay — APPROVED** [review]
Lgtm

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**